onboarding/src/Components/hooks/Packages.js

```diff
@@ -5,78 +5,132 @@ import PackagesRow from "../PackagesList/PackagesRow";
 /**
  * Get packages from Onboarding API when Packages component is loaded;
  */
-function Packages(props){
-	var [rows , setRows] = useState([]),
-		[loaded, isLoaded] = useState(false);
-	const [error, showError] = useState(null);
-	let url = getPath(),
-		fetchProps = {method:"GET", headers:{"Accept":"application/json", "Content-Type":"application/json", "X-CSRFToken":""}};
+function Packages(props) {
+  var [rows, setRows] = useState([]),
+    [loaded, isLoaded] = useState(false);
+  const [error, showError] = useState(null);
+  const [newRowId, setNewRowId] = useState(null);
 
-	useEffect(() => {
-		fetch(url + "api/package/", fetchProps).then(res => res.json()).then(
-			(result) => {
-				isLoaded(true);
-				setRows(result);
-			},
-			(error) => {
-				showError(error);
-				console.log(error);
-			}
-		);
-	}, [props.count]);
+  let url = getPath(),
+    fetchProps = {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        "X-CSRFToken": "",
```

Comment:
ma być puste?

Reply:
It seems that when you have a GET method
then you don't need token (I'm not sure why)